### PR TITLE
Allow unknown config properties

### PR DESCRIPTION
**Pull request by maxammann**
### Short description

Allows to parse configs that include unkown (potentially future) properties. Previouisly starting the server failed if you parse an unknown value.

### Side effects

None

### CODE REVIEW

**Review by f1sh1918 — APPROVED** [review]

File: backend/src/main/kotlin/app/ehrenamtskarte/backend/config/BackendConfiguration.kt
Comment (maxammann) [reply backend/src/main/kotlin/app/ehrenamtskarte/backend/config/BackendConfiguration.kt:73]: I added one. I think this should be permanent. See the comment for an explanation :)

Comment (maxammann) [reply backend/src/main/kotlin/app/ehrenamtskarte/backend/config/BackendConfiguration.kt:73]: If you agree feel free to click the merge button :)

Comment (maxammann) [reply backend/src/main/kotlin/app/ehrenamtskarte/backend/config/BackendConfiguration.kt:73]: @f1sh1918 what do you think?

Comment (f1sh1918) [reply backend/src/main/kotlin/app/ehrenamtskarte/backend/config/BackendConfiguration.kt:73]: i'm fine! Now we need two approvals :)

**Pull request merged**
